adapters/insticator/insticatortest/exemplary/app-video.json

```diff
@@ -25,14 +25,24 @@
         "publisher": {
           "id": "test-publisher-id"
         }
+      },
+      "device": {
+        "ipv6": "2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+        "ip": "1.22.22.1",
+        "ua": "Mozilla/5.0 (Linux; Android 8.0.0; SM-G960F Build/R16NW) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.84 Mobile Safari/537.36"
       }
     },
   
     "httpCalls": [
```

Comment:
Please verify the headers are set correctly in this test since some of them are derived from the device object. You can do so like this:
```
"httpCalls": [
    {
      "expectedRequest": {
        "headers": {
          "Content-Type": [
            "application/json;charset=utf-8"
          ],
          "Accept": [
            "application/json"
          ],
          etc...
``